pkg/api/validation/validation.go

```diff
@@ -1087,6 +1087,12 @@ func ValidatePersistentVolume(pv *api.PersistentVolume) field.ErrorList {
 	if numVolumes == 0 {
 		allErrs = append(allErrs, field.Required(specPath, "must specify a volume type"))
 	}
+
+	// do not allow hostPath mounts of '/' to have a 'recycle' reclaim policy
+	if pv.Spec.HostPath != nil && pv.Spec.HostPath.Path == "/" && pv.Spec.PersistentVolumeReclaimPolicy == api.PersistentVolumeReclaimRecycle {
```

Comment:
No question that it's a slippery slope -- I would be okay with adding `/etc`, `/var`, `/usr`, etc to a blacklist, but ultimately, I don't think you should be able to recycle a hostpath, period.  With that said, as I already said, it's v1 API, so we cannot really change it at this point, just try to limit harm.